### ISSUE TITLE
Doorways no longer render correctly (Rotating screen fixes visual issue)

### ISSUE THREAD

**Issue by Nicholas-Steel** [renaudbedard/fez-1.12-issues#156]
http://steamcommunity.com/sharedfiles/filedetails/?id=759070128
http://steamcommunity.com/sharedfiles/filedetails/?id=759070146

http://steamcommunity.com/sharedfiles/filedetails/?id=759070159
http://steamcommunity.com/sharedfiles/filedetails/?id=759070194

When entering the door, the door you just entered from will not appear correctly until the screen has been rotated at least once. This didn't occur when I last gave the game a complete playthrough which was a couple v1.12 builds ago.

I'm in a 100% NG+ playthrough and trying to find a a room with push blocks to check out their physics  to see how they've been modified/improved (but I can't find any rooms with the blocks)!

Nvidia Geforce 760 4GB with 372.54 drivers, Windows 10 x64 and Hardware Instancing & Multi-threading is Enabled.

**Comment by renaudbedard** [comment]
Huh, I was actually able to reproduce this with my 2nd PC but not my main development PC... Thanks for reporting, now I know I'm not crazy!

**Comment by Nicholas-Steel** [comment]
I think I might need to start a brand new playthrough to find a room with push blocks. it seems like every puzzle involving them causes them to either disappear or become inaccessible upon solving the puzzle.

**Comment by renaudbedard** [comment]
Yeah this is an annoying one because as much as I know it happens on your system, I can't get this to trigger at all here. I'll have to come back to it later. At least there's an easy workaround :)
